src/main/java/frc/robot/Constants.java

```diff
@@ -155,12 +155,28 @@ public static class TurretConstants {
     public static final int highGearCAN_CODER_ID = 0;
     public static final int lowGearCAN_CODER_ID = 0;
     public static final double ENCODER_ZERO_OFFSET_FROM_TURRET_ZERO_REV = 0;
+    public static final double GEAR_RATIO =
+        6.8333; // (24:164)  0.25 for the revbot prototype turret
+
+    // Simulation Vals
+    public static final double TURRET_MOI = 1; // TODO: Get real num // Moment of Inertia
+    public static final double TURRET_RADIUS = Units.inchesToMeters(12);
   }
 
   public static class OperatorConstants {
     public static final int kDriverControllerPort = 0;
   }
 
+  public static class SensorConstants {
+    // TODO: Get real ids
+    public static final int SOURCE_BREAK_ID = 1;
+    public static final int GROUND_BREAK_ID = 2;
+    public static final int BASE_BREAK_ID = 3;
+    public static final int SCORING_BREAK_ID = 4;
+
+    public static final double WRIST_BREAK_THOLD = 0.1;
```

Comment:
.1 mm is too small.  The note is 12 inches wide.  If we could the very peak of the torus, that would be about 6 inches away?  Can we start with that as a guess?